Assets/Scripts/Attributes/Animal.cs

```diff
@@ -97,6 +99,17 @@ public void InitDNA(DNA dna)
 
             if (sensors.FoundFleeTarget())
             {
+                if(sensors.FoundFood() || sensors.FoundWater())
+                {
+                    float distanceToPredator = DiffLength(sensors.GetFoundPredatorInfo().Position);
+                    bool shouldFlee = (ShouldFlee(HungerLimit, currentHunger, distanceToPredator) || ShouldFlee(ThirstLimit, currentThirst, distanceToPredator));
+                    if(shouldFlee)
+                        ChangeState(this.fleeState);
+                    else
+                    {
+                        HungerOrThirst();
+                    }
```

Comment:
Om man byter här skulle barn kunna gå in i hunger eller thirst state, vilket det inte är meningen att de ska göra. Det borde väl funka att inte ha denna else-satsen alls? Djuret flyr inte, och senare kollar den om den hittat mat eller vatten som vanligt. 

Annars skulle man kunna lägga till att den bara kollar efter mat och vatten om djuret är vuxet, typ `LookForFood(!isAdult() && ...)` etc.